home/home.css

```diff
@@ -0,0 +1,55 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+header {
+  padding: 13px;
+  margin: 72px 5px 10px 0px;
+}
+
+h1 {
+  font-size: 25px;
+  font-family: "Poppins", sans-serif;
+  font-weight: bold;
```

Comment:
instead of using tags name, It is better to use classes … to avoid overwrite or any other problem that may arise from the similarity of tags

Reply:
But every page has its own CSS File?